pkg/cmd/codespace/delete.go

```diff
@@ -178,6 +180,7 @@ func (a *App) Delete(ctx context.Context, opts deleteOptions) (err error) {
 					a.errLogger.Printf("error deleting codespace %q: %v\n", codespaceName, err)
 					return err
```

Comment:
This isn't your change, but I wonder if we shouldn't return the `err` here - the `errgroup.Wait` will return the first time that it sees an error, so it may either not attempt to delete all the codespaces, or it may exit event with some goroutines will running. Since we're already printing the error message, we could just `return nil` even if deletion fails just to ensure that all the deletes are attempted and complete

Reply:
Will need to check on this, I will get back.

Reply:
```
func (*errgroup.Group).Wait() error
Wait blocks until all function calls from the Go method have returned, then returns the first non-nil error (if any) from them.
```
It waits until all go routine has finished.

Reply:
Ahh good to know 😄 looks good then